src/components/Header.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Grid, Button } from "semantic-ui-react";
+import auth from '../modules/auth'
+
+const Header = (props) => {
+  const logOut = async () => {
+    try {
+      const response = await auth.signOut()
+      props.setAuthenticated()
```

Comment:
should `setAuthenticated()` include something from the response? Else we can remove `const response =`